### PR TITLE
[IMP] mail: introduce callPreview in public page

### PR DESCRIPTION
**Pull request by zel-odoo**
and sync the video/audio setting when joining a meeting

### CODE REVIEW

**Review by alexkuhn — COMMENTED** [review]
- `CallPreview` component not worth introducing it. Can just dump content in `WelcomePage`.
- Many indirections in the code. I have simplified it.

File: addons/mail/static/src/new/public/welcome_page.xml [hunk 1]
```diff
@@ -9,6 +9,25 @@
         </h1>
         <h2 class="m-5" t-esc="store.companyName"/>
         <div class="d-flex justify-content-center">
+            <!-- <div t-if="props.data.channelData.defaultDisplayMode === 'video_full_screen'" class="position-relative d-flex justify-content-center me-5" t-ref="root"> -->
```
Comment (seb-odoo) [inline addons/mail/static/src/new/public/welcome_page.xml:12]: comment should be removed?

Comment (zel-odoo) [reply addons/mail/static/src/new/public/welcome_page.xml:12]: https://github.com/odoo-dev/odoo/pull/2301

Comment (alexkuhn) [reply addons/mail/static/src/new/public/welcome_page.xml:12]: 🙇

**Pull request merged**